.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,8 +17,14 @@ Each of the questions and sections below start with multiple hash symbols (#). P
 
 ### Actual behavior:
 
+#### Actual NVDA speech/braille sample:
```

Comment:
I'm not sure that this always applies to bug reports. For instance, no (unexpected) speech/Braille is present when apps crash or hang or for developer-facing changes (an API works in an unexpected way, etc.)

Reply:
It can be left blank in those cases.
It might be worth indicating optional fields (e.g. including the "other questions", which are generally less relevant than expected speech, NVDA logs, etc)